1-JavaScript/Week2/homeworkSolition/ex6-sol-totalCost.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const cartForParty = {
+  beers: 1.75,
+  chips: 0.99,
+  chikenwings: 2.433,
+  water: 2,
+  bread: 1.5,
+};
+
+function calculateTotalPrice(cart) {
+  cart  = Object.values(cart)
+  const totalCartPrice = String(cart.reduce((acc,val)=>{return acc + val},0).toFixed(2))
+
+  return `Total: € ${totalCartPrice}`
+}
```

Comment:
Fantastic! 🤩 Just one more comment: I would not convert the total price into a String because 1) you might want to do some calculations with it and 2) if you use it in a template literal then it already gets converted so you don't need to use `String`.

```suggestion
function calculateTotalPrice(cart) {
  cart  = Object.values(cart)
  const totalCartPrice = cart.reduce((acc,val)=>{return acc + val},0).toFixed(2)

  return `Total: € ${totalCartPrice}`
}
```